addons/graphql/src/components/FullScreen/style.ts

```diff
@@ -1,6 +1,6 @@
-export default {
+export const style = {
   wrapper: {
-    position: 'absolute',
+    position: 'absolute' as 'absolute',
```

Comment:
You can use `const` assertions, see [TypeScript 3.4 Release Note](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html): `position: 'absolute' as const,`

Reply:
I saw that on other migrations here(yours) and didn't catch lastest typescript and applied what I alreay knew... shame on me. I applied it!